testsuite/tests/workflows/air-gapping/my_index/crates/hello/HEAD

```diff
@@ -0,0 +1 @@
+ref: refs/heads/master
```

Comment:
I remember having some troubles in the past because of bare repositories within the parent repository. If it's the same to you, I'd rather prefer that the crates are .tgz files and you use `tar` as the mock command.

Reply:
Sure, no problem.